ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/AvatarMPCControllerThread.java

```diff
@@ -0,0 +1,409 @@
+package us.ihmc.avatar;
```

Comment:
Is there a reason that this is in open robotics and not in the crocoddyl repo?

Reply:
It doesn't have much meaning. I just put this class to the place where AvatarControllerThread is. 
Do you think moving this class into crocoddyl repo is better?

Reply:
Got it. I saw the below comment now,
I agree with you. 
I would move all the mpc specific classes to the crocoddyl repo.
 < we talked in person >